packages/@aws-cdk/aws-lambda/test/integ.layer-version.lit.ts

```diff
@@ -12,7 +12,7 @@ const awsAccountId = stack.account;
 /// !show
 const layer = new lambda.LayerVersion(stack, 'MyLayer', {
   code: lambda.Code.fromAsset(path.join(__dirname, 'layer-code')),
-  compatibleRuntimes: [lambda.Runtime.NODEJS_10_X],
+  compatibleRuntimes: [lambda.Runtime.NODEJS_14_X],
```

Comment:
got an error when trying to update the integration test, saying that nodejs_10_x was not supported any more